_layouts/lesson.html

```diff
@@ -28,7 +28,7 @@
 
       <div class="col-md-8">
         <div class="header-title">
-          <h1><a href="{{ site.baseurl }}/{{ page.url }}">{{ page.title }}</a></h1>
+          <h1><a href="{{ site.url }}{{ page.url }}">{{ page.title }}</a></h1>
```

Comment:
I'd prefer this to be a relative URL instead of absolute. It's odd that `site.baseurl` doesn't map correctly... but anyway, can you change this to:

``` html
<h1><a href="/{{ page.url }}">{{ page.title }}</a></h1>
```